code/src/index.css

```diff
@@ -11,3 +12,95 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  max-width: 100vw;
+  background-color: #ff4e00;
+  background-image: linear-gradient(315deg, #ff4e00 0%, #ec9f05 74%);
+  border-radius: 10%;
+  padding: 5vw;
+}
+
+.form {
+  margin-left: auto;
+  margin-right: auto;
+  
+}
+
+/*QUESTION*/
+
+.alternative-container {
+  padding: 3px;
+}
+
+
+select {
+  position: relative;
+  display: inline-block;
+  margin-bottom: 15px;
+  width: 100%;
+}    
+select {
+      display: inline-block;
+      width: inherit;
+      cursor: pointer;
+      padding: 10px 15px;
+      outline: 0;
+      border: 0px solid #000000;
+      border-radius: 10px;
+      background: #e6e6e6;
+      color: #7b7b7b;
+  }
```

Comment:
I´m a little confused by "select" being assigned properties twice here, is this intentional? If so, is it used to create something in a certain manner that I didn´t know about - curious to know 😀  I´m thinking now one maybe is overwriting the other?